wazuh/wazuh-dashboard-plugins
# [Vulnerabilities dashboards] Fix vulnerability dashboard filters

### Description
This PR, fixes a bug in the filters of the vulnerabilities dashboard when a filter is applied in the search bar.
 
### Issues Resolved
Closes #6064 

### Evidence
[Evidence.webm](https://github.com/wazuh/wazuh-dashboard-plugins/assets/43619595/44bb461a-f251-43db-828d-f8f68695da54)


### Test
1. Navigate to Modules -> Vulnerabilities
2. Click on an option in a filter on the dashboard
3. See if filter options change based on the added filter

### Check List
